app/Models/Token.ts

```diff
@@ -24,6 +24,12 @@ export default class Token extends BaseModel {
   @column()
   public is_revoked: boolean;
 
+  @column()
+  public name: string;
+
+  @column.dateTime()
+  public expires_at: DateTime;
```

Comment:
Why did you add these new fields? How are they going to be backwards compatible?

Reply:
That is a great question. I'm not entirely sure if it is ok to add them like so or if we need to provide a @default value. I've never used Adonis before so I'm not sure how to handle it.

If you test the auth jwt or even the api you will noticed that this 2 fields were added in the Adonis upgrade.

Generally, we have a bigger problem than this... I've tried to test the db in production and it fails to even initiate migrations (even without this new migration I added). We will probably need to code a script to migrate old databases to the new database schema manually